src/node_crypto.cc

```diff
@@ -5672,6 +5693,11 @@ void Initialize(Local<Object> target,
   env->SetMethodNoSideEffect(target, "certVerifySpkac", VerifySpkac);
   env->SetMethodNoSideEffect(target, "certExportPublicKey", ExportPublicKey);
   env->SetMethodNoSideEffect(target, "certExportChallenge", ExportChallenge);
+  env->SetMethodNoSideEffect(target, "certLoadExtraRootCertsFile",
+    LoadExtraRootCertsFile);
+  // Exposed for testing purposes only.
+  env->SetMethodNoSideEffect(target, "certIsExtraRootCertsFileLoaded",
+    IsExtraRootCertsFileLoaded);
```

Comment:
The `certIsExtra...` name is kind of misleading; the other `cert...` methods are for SPKAC.